pkg/mcs/tso/server/server.go

```diff
@@ -122,36 +120,11 @@ func CreateServerWrapper(cmd *cobra.Command, args []string) {
 	metricutil.Push(&cfg.Metric)
 
 	// TODO: Create the server
```

Comment:
Please ignore the hack. Working on a separate pr to init/start tso and will rewrite this function.